src/common/header/header.scss

```diff
@@ -0,0 +1,466 @@
10.281 0 0 0-2.214 1.188l-1.107-2.3a10.445 10.445 0 0 1 5.886-1.809 6.9 6.9 0 0 1 4.739 1.5 5.491 5.491 0 0 1 1.688 4.334v4.4a1.271 1.271 0 0 0 .2.81.887.887 0 0 0 .661.27V0a8.055 8.055 0 0 1-1.593.189A2.412 2.412 0 0 1 163.5-.27a1.973 1.973 0 0 1-.689-1.215l-.081-.783a6.262 6.262 0 0 1-2.295 1.89A6.59 6.59 0 0 1 157.545.27Zm1.026-2.646a4.713 4.713 0 0 0 1.742-.324 3.161 3.161 0 0 0 1.283-.864 1.3 1.3 0 0 0 .594-1.026V-6.21a8.858 8.858 0 0 0-1.4-.392 7.562 7.562 0 0 0-1.458-.148 3.865 3.865 0 0 0-2.295.635 1.905 1.905 0 0 0-.891 1.606 1.933 1.933 0 0 0 .7 1.526A2.548 2.548 0 0 0 158.571-2.376Zm10.557 5.13a6.98 6.98 0 0 0 .891.2 4.855 4.855 0 0 0 .729.068 1.548 1.548 0 0 0 .648-.135 1.388 1.388 0 0 0 .54-.473 4.613 4.613 0 0 0 .486-.931q.243-.594.513-1.485l-5.589-14.148h3.726l3.834 11.016 3.4-11.016h3.4L175.77 2.808a5.066 5.066 0 0 1-1.755 2.43 4.935 4.935 0 0 1-3.078.945 5.852 5.852 0 0 1-.878-.068 5.611 5.611 0 0 1-.932-.229Z' transform='translate(3 33.587)' fill='%23fff'/%3E%3Cg id='Group_5' data-name='Group 5' transform='translate(83.211)'%3E%3Cg id='Group_10' data-name='Group 10'%3E%3Cg id='Ellipse_8' data-name='Ellipse 8' transform='translate(18.197 0) rotate(60)' fill='none' stroke='%2300f2fe' stroke-linecap='round' stroke-width='2.6'%3E%3Cellipse cx='20.579' cy='8.689' rx='20.579' ry='8.689' stroke='none'/%3E%3Cellipse cx='20.579' cy='8.689' rx='19.279' ry='7.389' fill='none'/%3E%3C/g%3E%3Cg id='Ellipse_9' data-name='Ellipse 9' transform='translate(-0.211 13.587)' fill='none' stroke='%2300f2fe' stroke-linecap='round' stroke-width='2.6'%3E%3Cellipse cx='20.5' cy='8.5' rx='20.5' ry='8.5' stroke='none'/%3E%3Cellipse cx='20.5' cy='8.5' rx='19.2' ry='7.2' fill='none'/%3E%3C/g%3E%3Cg id='Ellipse_10' data-name='Ellipse 10' transform='translate(38.776 8.689) rotate(120)' fill='none' stroke='%2300f2fe' stroke-linecap='round' stroke-width='2.6'%3E%3Cellipse cx='20.579' cy='8.689' rx='20.579' ry='8.689' stroke='none'/%3E%3Cellipse cx='20.579' cy='8.689' rx='19.279' ry='7.389' fill='none'/%3E%3C/g%3E%3C/g%3E%3C/g%3E%3C/g%3E%3Cpath id='Path_1' data-name='Path 1' d='M408.4 30.28l-4.563 7.985 4.563 6.274h7.7l4.706-6.844L416.81 30.28Z' transform='translate(-11.246 -2.736)' fill='%2361dafb'/%3E%3Cpath id='Polygon_2' data-name='Polygon 2' d='M26.2 12.227a2.645 2.645 0 0 1 4.523 2.6l-.018.048a2.645 2.645 0 0 0 2.04 3.529l.087.015a2.645 2.645 0 0 1 0 5.216l-.087.015a2.645 2.645 0 0 0-2.04 3.529l.018.048a2.645 2.645 0 0 1-4.523 2.6h0a2.645 2.645 0 0 0-4.089 0h0a2.645 2.645 0 0 1-4.523-2.6l.018-.048a2.645 2.645 0 0 0-2.04-3.529l-.087-.015a2.645 2.645 0 0 1 0-5.216l.087-.015a2.645 2.645 0 0 0 2.04-3.529l-.018-.048a2.645 2.645 0 0 1 4.523-2.6h0a2.645 2.645 0 0 0 4.089 0Z' transform='translate(390.665 4.291) rotate(30)' fill='%2300f2fe'/%3E%3Cpath id='Polygon_3' data-name='Polygon 3' d='M5.689 1.122a1 1 0 0 1 1.621 0l4.544 6.292A1 1 0 0 1 11.044 9H1.956a1 1 0 0 1-.811-1.585Z' transform='translate(407 27.544) rotate(90)' fill='%23010326'/%3E%3C/g%3E%3C/svg%3E"); */
```

Comment:
@6km , the commented code can be removed.

Reply:
done